auctions/english-auction/src/data_structures/nft_asset.sw

```diff
@@ -0,0 +1,61 @@
+library nft_asset;
+
+dep traits;
+
+use ::errors::AssetError;
+use traits::Asset;
+
+pub struct NFTAsset {
+    /// The `ContractId` of the NFT that the struct is representing.
+    contract_id: ContractId,
```

Comment:
`contract_id` is a module so we need to be mindful of the prelude and possible namespace collisions / shadowing